### PR TITLE
Task/bump deps

### PR DESCRIPTION
**Pull request by solomontang**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,29 +70,29 @@
     "babel-loader": "~8.0.0",
```
Comment (UnsungHero97) [inline package.json:70]: `~8.1.0`

**Comment by solomontang** [comment]
peer deps should probably just be left using ^ unless there's some breaking change between interdependencies

**Comment by UnsungHero97** [comment]
> peer deps should probably just be left using ^ unless there's some breaking change between interdependencies

I mean, why not bump them to match the regular dependencies? i.e.

```
"@babel/runtime": "~7.9.0",
"react": "^16.13.0",
"react-dom": "^16.13.0",
```

**Comment by solomontang** [comment]
i guess it would force people to bump?

**Pull request merged**